index.js

```diff
@@ -1 +1,76 @@
-// Your code here
+function createEmployeeRecord(employee) {
+  const obj = {
+    firstName: employee[0],
+    familyName: employee[1],
+    title: employee[2],
+    payPerHour: employee[3],
+    timeInEvents: [],
+    timeOutEvents: []
+  }
+  return obj
+}
+
+function createEmployeeRecords(employeeData) {
+  const employeeRecords = []
+  employeeData.forEach(employee => {
+    const record = createEmployeeRecord(employee)
+    employeeRecords.push(record)
+  })
+  return employeeRecords
```

Comment:
This is exactly what `map` was built for:
```suggestion
  return employeeData.map(employee => createEmployeeRecord(employee))
```